src/client.vala

```diff
@@ -52,15 +52,18 @@ private void on_subscribe (uint count, bool dnd, bool cc_open) {
 
 private void on_subscribe_waybar (uint count, bool dnd, bool cc_open) {
     string state = (dnd ? "dnd-" : "") + (count > 0 ? "notification" : "none");
-    if (cc_open) state += " cc-open";
 
     string tooltip = "";
     if (count > 0) {
         tooltip = @"$(count) Notification" + (count > 1 ? "s" : "");
     }
-
-    print ("{\"text\": \"\", \"alt\": \"%s\", \"tooltip\": \"%s\", \"class\": \"%s\"}\n",
-           state, tooltip, state);
+    print ("{\"text\": \"\", \"alt\": \"%s\", \"tooltip\": \"%s\", \"class\": ",state, tooltip);
+    if (cc_open) {
+        print("[\"%s\", \"cc-open\"]", state);
```

Comment:
It would be probably nicer to build up a Vala array of classes, and then convert that array to JSON in one step.
However I'm not too familiar with Vala so I did it this way with an `if-else` for now